python/pyspark/sql/functions.py

```diff
@@ -2181,31 +2186,69 @@ def udf(f=None, returnType=StringType()):
 @since(2.3)
 def pandas_udf(f=None, returnType=StringType()):
     """
-    Creates a :class:`Column` expression representing a user defined function (UDF) that accepts
-    `Pandas.Series` as input arguments and outputs a `Pandas.Series` of the same length.
+    Creates a :class:`Column` expression representing a vectorized user defined function (UDF).
+
+    The user-defined function can define one of the following transformations:
+    1. One or more `pandas.Series` -> A `pandas.Series`
```

Comment:
Looks we should leave a newline for python doc.

Reply:
Don't forget to note that the output `Pandas.Series` should be the same length.

Reply:
Fixed. Add the note about length.